test/integration/suites/upgrade/05-check-workload-svid

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -o pipefail
```

Comment:
Hmm, this isn't strictly needed anymore (i was doing some piping in an earlier revision). If the api fetch fails, it will result in the "fail-now" happening.